packages/config/src/api/site_info.ts

```diff
@@ -55,7 +62,7 @@ export const getSiteInfo = async function ({
     getSite(api, siteId, siteFeatureFlagPrefix),
     getAccounts(api),
     getAddons(api, siteId),
-    getIntegrations({ siteId, testOpts, offline, accountId }),
+    getIntegrations({ siteId, testOpts, offline, accountId, token, sendBuildBotTokenToJigsaw }),
```

Comment:
Perhaps we pass along the whole `featureFlags` object down the chain?
Good or bad aside, this appears to be the pattern I'm seeing being used in general in this repo. I'm also thinking of the situation where you need to add another flag in for some reason, passing the object down means you don't need to make as many changes.